.circleci/config.yml

```diff
@@ -349,10 +401,10 @@ jobs:
 
   check-kontrol-build:
     docker:
-      - image: <<pipeline.parameters.ci_builder_image>>
+      - image: cimg/base:2024.01
```

Comment:
nit: Can we still use a param for this? Otherwise we'll wind up with different images used and different versions over time.

Reply:
yes - if you don't mind, i'll do that as a follow-up since i've already switched the executors back to free up the pipeline. i'll merge in the AM then refactor to use a param.